osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -305,6 +312,9 @@ public override void OnSuspending(ScreenTransitionEvent e)
         {
             base.OnSuspending(e);
 
+            quickRestartBlackLayer?.FadeOut(500, Easing.OutQuint);
```

Comment:
Missing expire call? I'd expect one given the only reference to this box is being dropped here.

```suggestion
            quickRestartBlackLayer?.FadeOut(500, Easing.OutQuint).Expire();
```